README.md

```diff
@@ -1,2 +1,80 @@
-# weather-app
-This is a weather app that lets users search and get updated with the weather info of a particular place.
+# Weather App ☁️
+
+## Description
+
+Weather app is a web application that shows the current weather information about a given location.
```

Comment:
- [ ] It can be `The Weather app is a web application that shows the current weather information of a given location.`

Reply:
I will take this as an optional suggestion, thank you.